.eslintignore

```diff
@@ -1,4 +0,0 @@
-/node_modules
```

Comment:
I think we should either revert this file or add the exclusions to an `ignorePatterns` item in `.eslintrc.js`.

Reply:
It is using the .gitignore which seems like good practice to avoid unnecessary config:

https://github.com/AqoviaElements/stacked-sheets/blob/86d9469bfc3100f89efb3e15848f16453828a335/package.json#L14